1_fetch.R

```diff
@@ -73,13 +73,17 @@ p1_targets_list <- list(
   ),
   
   # Dissolve NHDPlusv2 catchments to create a single catchment polygon for
-  # each NHM segment (analogous to HRU).
+  # each NHM segment (analogous to HRU). Note that we are aggregating by
+  # seg_id_nat, so split subsegid's (e.g. 3_1, 3_2; 8_1, 8_2; 152_1, 152_2)
+  # will be combined. We end up with 455 catchments instead of 456 because
+  # one very short segment, subsegid 287_1/seg_id_nat 1721, does not have an 
+  # NHDPlusv2 catchment.
   tar_target(
     p1_nhm_catchments_dissolved,
-    left_join(p1_nhd_catchments %>% mutate(COMID = as.character(COMID)),
-              p1_drb_comids_all_tribs %>% mutate(COMID = as.character(COMID)),
+    left_join(x = mutate(p1_nhd_catchments, COMID = as.character(COMID)),
```

Comment:
There's no efficiency improvement. I'm definitely a "pipe everywhere" person! I made these edits for legibility because I think if there's only one step (e.g. `mutate`), then the non-piped version is easier for some folks to interpret (I've seen this style used more in some repos I've been working in recently). But it does not make a difference for the output.